### ISSUE TITLE
FAQ. Why does BackArch use separate repository?

### ISSUE THREAD

**Issue by ejiektpobehuk** [BlackArch/blackarch-site#125]
This is a question from a regular Archlinux user perspective.

I was looking for a tool in Arch repos with pacman.
Not there.

I took a look at AUR - not there.

BlackArch - it's there!

First of all thanks for packaging such a huge variety of awesome tools!
Nevertheless I feel that there is room for improvement in terms of sharing it with ArchLinux.
I understand why there is a distro - it's preconfigured for a particular penetration related purpose, but I don't see a clear reason for staying out of `[commutiny]` repo.
Or at least spreading your packages over AUR (to automatically push to git subtree).

My proposal is to state in FAQ section why separate repo is necessary or come up with actionable items to bring.

Thanks!

**Comment by noraj** [comment]
Hi,

You can add the BlackArch (BA) repository [on top of ArchLinux](https://blackarch.org/downloads.html#install-repo) (AL).
So you can already see it as a community repository but not managed by AL guys.
Sync to AUR is a lot of extra work for no gain, having a repository is 10 times more handy than tons of separate package on AUR.
The main reason why BA is not a "security" repo on AL is because we work in a very different way than AL. BA is more bleeding edge, lot of git package, less staging and testing, ...

**Comment by ejiektpobehuk** [comment]
Thanks for the response!

The gain of having it closer to AL users is discoverability.
I don't suggest to handle all of AUR packages manually but rather set up an git subtree based automation.
What's bad in GitHub Action pushing stuff to AUR where it's possible to mention that package is originating from particular GitHub repo and available in BA.
Is it a hassle to setup initially - yes.
Is it any good - definitely (more people get access to awesome tools and BA becomes a little bit more popular).
Is it good enough - I afraid you're right, not enough.

To sum up.
I'm satisfied with this answer but I just would like it to be more user facing.
